libnetdata/string/utf8.c

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include "../libnetdata.h"
+
+/**
+ * Get byte length
+ *
+ * Measure the utf8 length
+ *
+ * @param c is the utf8 character
+ *  *
+ * @return It reurns the length of the specific character.
+ */
+char url_utf8_get_byte_length(char c) {
```

Comment:
should we keep `url` prefix?

Reply:
I am keeping the name in this PR, because it is used inside url library and I did not want to changes files that does not have direct relationship with labels, but I agree that it is necessary to have another PR renaming it.